plugins/image-prioritizer/tests/test-helper.php

```diff
@@ -102,13 +102,6 @@ public function test_end_to_end( string $directory ): void {
 	 * @return array<string, mixed> Data.
 	 */
 	public function data_provider_test_auto_sizes(): array {
-		$outside_viewport_rect = array_merge(
-			$this->get_sample_dom_rect(),
-			array(
-				'top' => 1000,
-			)
-		);
-
```

Comment:
This is removed because now in `plugins/optimization-detective/tests/class-optimization-detective-test-helpers.php` it will handle sparse `boundingClientRect` by merging on top of a full one.

Reply:
The other changes similar changes in this PR are for the same reason.